src/test/java/com/mig82/folders/FolderPropertiesTest.java

```diff
@@ -78,16 +76,24 @@ public void testFreestyleInSubFolder() throws Exception {
     }
 
     @Test
-    public void testPipelineInNode() throws Exception {
+    void testPipelineInSubFolder(TestInfo info) throws Exception {
+        // Create a subfolder.
+        Folder sub = f.createProject(Folder.class, "sub-2");
 
-        // Create a pipeline job which uses the properties from its parent folder.
+        // Add a property in the subfolder that overrides another in the parent folder.
+        FolderProperties<?> properties = new FolderProperties<>();
+        properties.setProperties(new StringProperty[] {new StringProperty("key1", "override")});
+        sub.addProperty(properties);
+
+        // Create a pipeline job inside the subfolder which uses the properties from its parent folder.
         WorkflowJob p = PipelineTestHelper.createJob(
-                f,
-                "p-" + name.getMethodName(),
+                sub,
+                "p-" + info.getTestMethod().orElseThrow().getName(),
                 """
                 node {
-                  wrap([$class: 'ParentFolderBuildWrapper']) {
+                  wrap([$class: 'ParentFolderBuildWrapper']){
```

Comment:
Seems like an unnecessary change in spacing.
```suggestion
                  wrap([$class: 'ParentFolderBuildWrapper']) {
```